environments/production/public.yml

```
@@ -158,7 +158,7 @@ localsettings:
   ENABLE_SOFT_ASSERT_EMAILS: True
   ENFORCE_SSO_LOGIN: True
   ELASTICSEARCH_MAJOR_VERSION: 2
-  ES_CASE_SEARCH_INDEX_NAME: "case_search_2021-06-04"
+  ES_CASE_SEARCH_INDEX_NAME: "case_search_2022-10-14"
```

I just checked and the name on prod is actually `case_search_2022-10-04`
```suggestion
  ES_CASE_SEARCH_INDEX_NAME: "case_search_2022-10-04"
```